src/components/accordion.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from "@lingui/macro";
+import React from "react";
+
+export const Accordion = (props: {
+  question: string;
+  children: React.ReactNode;
+  /** When set to true, the accordion is open on initial view. */
+  isExpanded?: boolean;
+}) => (
+  <div className="jf-accordion-item">
+    <details className="has-text-left" open={props.isExpanded}>
+      <summary>
```

Comment:
FYI, @austensen idk if you have experience working with the [<details> HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) but I'm obsessed. It's a fully accessible, JS-free way to make an interactive, expandable menu. I just learned what it was only a little while ago.